src/App.js

```diff
@@ -1,25 +1,33 @@
-import logo from './logo.svg';
-import './App.css';
+// src/Greeting.js
+import React, { useEffect, useState } from 'react';
+
+const Greeting = () => {
+  const [greeting, setGreeting] = useState('');
+
+  useEffect(() => {
+    const fetchGreeting = async () => {
+      try {
+        const response = await fetch('http://localhost:3001/random_greeting');
+        const data = await response.json();
+        setGreeting(data.greeting); // Assuming the API response has a 'greeting' property
+      } catch (error) {
+        console.log(error);
+      }
+    };
+
+    // Call the fetchGreeting function
+    fetchGreeting();
+  }, []); // The empty dependency array ensures that the effect runs only once
```

Comment:
Nice job so far👍🏼, I noticed you already have a greeting component, and you are creating another greeting class in the supposed app component. Just as required, you should have a greeting component and also an `app` component with routes from `react-router` to navigate through your greeting components to render the greetings😉.

<img width="816" alt="image" src="https://github.com/SaveryIV/hello-react-front-end/assets/64643995/e2fb83c2-baa4-43e0-bfbb-44d5fc04ebf4">